draft-ietf-rtcweb-jsep.xml

```diff
@@ -1713,58 +1769,37 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-          the negotiated session descriptions by following the steps mentioned
-          for the "local-offer" state above, along with these exceptions:
-          [OPEN ISSUE: should this be permitted in the remote-pranswer state?
-          https://github.com/rtcweb-wg/jsep/issues/143]
-          <list style="symbols">
-              <t>If a m= section exists in the current local description, but
-              does not have an associated local MediaStreamTrack (possibly
-              because said MediaStreamTrack was removed since the last
-              exchange), a m= section MUST still be generated in the new offer,
-              as indicated in <xref target="RFC3264"></xref>, Section 8. The
-              disposition of this section will depend on the state of the
-              remote MediaStreamTrack associated with this m= section. If one
-              exists, and it is still in the "live" state, the new m= section
-              MUST be marked as "a=recvonly", with no "a=msid" or related
-              attributes present. If no remote MediaStreamTrack
-              exists, or it is in the "ended" state, the m= section MUST be
-              marked as rejected, by setting the port to zero, as indicated in
-              <xref target="RFC3264"></xref>, Section 8.2.</t>
-
-              <t>If any MediaStreamTracks have been added, and there exist
-              recvonly m= sections of the appropriate media type with no
-              associated MediaStreamTracks, or rejected m= sections of any
-              media type, those m= sections MUST be recycled, and a
-              local MediaStreamTrack associated with these recycled m=
-              sections until all such existing m= sections have been
-              used. This includes any recvonly or rejected m= sections
-              created by the preceding paragraph.</t>
+              <t>For RtpTransceivers with an RtpSender that is still
+              active, the "a=msid", "a=ssrc", and "a=ssrc-group" lines
+              MUST stay the same.</t>
+
+              <t>If any RtpTransceiver has had its RtpSender or
+              RtpReceiver activated or deactivated, its m=
+              section MUST be marked so by changing the value of the
+              <xref target="RFC3264"></xref> directional attribute to
+              "a=sendrecv" if both are active, "a=sendonly" if only
+              the sender is active, "a=recvonly" if the only the
+              receiver is active, and "a=inactive" if both are
+              inactive.</t>
+
+              <t>If any RtpTransceivers has been stopped, its m=
+              section MUST be marked rejected by setting the port to
+              zero and removing the "a=msid", "a=ssrc", and
+              "a=ssrc-group" lines.</t>
+
+              <t>If any RtpTransceiver has been added, and there
+              exists an m= section of the appropriate media type with
+              no associated RtpTransceivers (i.e. as described in the
```

Comment:
I didn't grok the reference to the preceding paragraph. Better to just say stopped?